docker/centos7.3/files/image-runtime-scripts/start-services-script.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+#### Generate input for Ambari from container_limits generated dynamically by YARN
+echo "using container_limits to create system resource overrides for ambari"
+memoryMb=`cat container_limits | grep memory= | awk -F= '{print $2}'`
```

Comment:
```sh
memoryMb=`awk -F= '/memory=/{print $2}' container_limits`
```

Reply:
I'd like to stick to the original base here